src/odemis/acq/stitching/_registrar.py

```diff
@@ -54,48 +55,54 @@ def addTile(self,tile,dependent_tiles=None):
             for t in dependent_tiles:
                 pos.append(t.metadata[model.MD_POS])
             self.dep_tiles_pos.append(pos)
-                
+
     def getPositions(self):
         """ 
         returns:
         tile_positions (list of N tuples of 2 floats)): the adjusted position in X/Y for each tile, in the order they were added
         dep_tile_positions (list of N tuples of tuples of 2 floats): the adjusted position for each dependent tile 
         (in the order they were passed)
-        """       
+        """
         return self.tile_pos, self.dep_tiles_pos
-        
-        
+
+
 class ShiftRegistrar(object):
     """ 
     Locates the position of the image relative to the previous top tiles (horizontally and vertically) 
     by using cross-correlation. The cross-correlation is done using just the part of the images which are 
     supposed to be overlapping. In case the cross-correlation doesn't work (based on a couple of simple tests), 
     fallback to the average shift on the same axis. 
     """
-    
+
     def __init__(self):
-        # arrays to store the vertical/horizontal shift values measured for each tile
-        self.nx = 1 # initialize grid to 1x1. The size will increase as new tiles are added.
+        # arrays to store the vertical/horizontal shift values measured for
+        # each tile
+        # initialize grid to 1x1. The size will increase as new tiles are
+        # added.
+        self.nx = 1
         self.ny = 1
-        self.coords_ver = [[None for _ in range(self.nx)] for _ in range(self.ny)] 
-        self.coords_hor = [[None for _ in range(self.nx)] for _ in range(self.ny)]
-        self.shifts = [[(0, 0) for _ in range(self.nx)] for _ in range(self.ny)]
+        self.coords_ver = [
```

Comment:
These 3 lines should stay on a single line each, it's more readable in such a way (even if it's a tiny bit longer than 80 characters).